packages/manager/apps/anthos/package.json

```diff
@@ -19,6 +19,7 @@
     "start:watch": "lerna exec --stream --parallel --scope='@ovh-ux/manager-anthos-app' --include-dependencies -- npm run dev:watch --if-present"
   },
   "dependencies": {
+    "@ovh-ux/manager-billing-components": "^2.0.0 || ^3.0.0",
```

Comment:
According to `@ovh-ux/manager-billing-components`(https://github.com/ovh/manager/blob/master/packages/manager/modules/billing-components/package.json#L22)  following dependencies are required :
- `@ovh-ux/ng-at-internet`
- `@ovh-ux/ng-ovh-feature-flipping`
- `@ovh-ux/ng-ovh-payment-method`
- `bootstrap`
- `moment`
which are not present currently in this app so it is not standalone and probably not working properly

Reply:
Probably, we will have to come up with an automation to generate the dependencies... Its too hard to track them, especially, when they are multi-leveled

Reply:
True, done.